lib/internal/process/per_thread.js

```diff
@@ -48,6 +48,10 @@ const {
 } = require('internal/validators');
 const constants = internalBinding('constants').os.signals;
 
+const {
+  handleProcessExit,
+} = require('internal/modules/esm/handle_process_exit');
```

Comment:
If this logic happens for all code, not just ESM code, should `handle_process_exit.js` be under `modules/esm`?

Reply:
It only happens for ESM code, because it's the only place we have TLA in Node.js.

Reply:
I am curious what's the full name of TLA? 🤣

Reply:
It's an acronym for Top Level Await, sorry I was too lazy to type it all out 🙈